samples/snippets/fsharp/VS_Snippets_CLR/Array_SortComparison/fs/source.fs

```diff
@@ -0,0 +1,63 @@
+﻿// <Snippet1>
+open System
+
+let compareDinosByLength (x: string) (y: string) =
+    if isNull x then 

```

Comment:
Generally in F# we use
```
    match x with 
    | null -> ...
    | _ -> ...
```

However I also wonder if the example should show `null` at all - it may depend on the text of the article - in F# using `null` strings is very very rare. Still, I understand why you have it here.